### ISSUE TITLE
[BUG] Latest govulncheck breaks action

### ISSUE THREAD

**Issue by Templum** [Templum/govulncheck-action#31]
**Setting a Baseline**

Logs:

```
2:13PM INF GoEnvironment Details: GOPRIVATE=github.com/Templum/private-lib Go-Arch=amd64 Go-Os=linux Go-Version=go1.20
2:13PM INF Running govulncheck for package ./... in dir /github/workspace
2:14PM ERR parsing govulncheck output yielded error error="json: cannot unmarshal array into Go value of type types.Result"
2:14PM ERR Scanning yielded error error="scan failed to produce proper report"
```

**Bug Description**
It appears that within the last [release](https://pkg.go.dev/golang.org/x/vuln@v0.0.0-20230320232729-bfc1eaef17a4/cmd/govulncheck?tab=versions) from yesterday some changes to the JSON format where performed.

**Comment by Templum** [comment]
As far as I know there is no publicly available code covering the JSON Report format, I will need to verify this. But in the meanwhile, I decided to go with the approach where I set the default version to the last known working version.

Which will be `v0.0.0-20230320232729-bfc1eaef17a4`.

**Comment by Templum** [comment]
I'm sorry for any inconvenience this might have caused. It was released in version v0.10.0 of the action and should be available starting now.
